src/py/crankshaft/crankshaft/query_runner.py

```diff
@@ -0,0 +1,43 @@
+"""class for fetching data"""
+import plpy
+
+
+class QueryRunner:
```

Comment:
As this is attached to moran/markov queries, I would go with a name closer to that. QueryRunner is quite generic, while get_markov is very specific. What about something like `AnalysisDataProvider`?